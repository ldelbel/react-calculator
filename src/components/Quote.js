import Navbar from './Navbar';

const styles1 = {
  display: 'flex',
  flexDirection: 'column',
  justifyContent: 'center',
  alignItems: 'center',
  fontSize: '2.5em',
  paddingTop: '120px'
}

const Quote = () => {
  return (
    <>
      <Navbar />
      <main style={styles1}>
        <div style={{width: '80%'}}>
          <p>
            "Mathematics is the most beautiful and most powerful creation of the human spirit."
          </p>
          <p>
            - Stefan Banach
          </p>
        </div>
      </main>
    </>
  )
}

export default Quote;