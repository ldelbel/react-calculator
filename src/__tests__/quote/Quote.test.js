import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Quote from '../../components/Quote';

it('renders correctly', () => {
  const quote = renderer.create(
    <BrowserRouter>
      <Quote />
    </BrowserRouter>,
  ).toJSON();
  expect(quote).toMatchSnapshot();
});
