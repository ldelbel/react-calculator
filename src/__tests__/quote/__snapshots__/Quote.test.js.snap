// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <nav
    className="sc-bdfBwQ hjshVr"
  >
    <img
      alt="img"
      src="math.png"
    />
    <div>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        href="/calc"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </div>
  </nav>,
  <main
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "fontSize": "2.5em",
        "justifyContent": "center",
        "paddingTop": "120px",
      }
    }
  >
    <div
      style={
        Object {
          "width": "80%",
        }
      }
    >
      <p>
        "Mathematics is the most beautiful and most powerful creation of the human spirit."
      </p>
      <p>
        - Stefan Banach
      </p>
    </div>
  </main>,
]
`;
